lib/tasks/subscriptions/test.rake

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+namespace :ofn do
+  namespace :subs do
+    namespace :test do
+      desc "Repeat placement job for a specific Order Cycle"
+      task repeat_placement_job: :environment do
+        puts "WARNING: this task will generate new, and potentially duplicate, customer orders"
+        exit_in_production
+
+        order_cycle_id = request_order_cycle_id
+
+        # Open Order Cycle by moving close_at to the future and open_at to the past
+        set_order_cycle_times(
+          order_cycle_id,
+          15.minutes.ago,
+          15.minutes.from_now
+        )
+
+        # Reset Proxy Orders of the Order Cycle
+        #   by detatching them from existing orders and resetting placed and confirmed dates
+        ProxyOrder.find_by(order_cycle_id: order_cycle_id)&.update!(
+          order_id: nil,
+          confirmed_at: nil,
+          placed_at: nil)
+
+        # Run placement job to create orders
+        SubscriptionPlacementJob.new.perform
+      end
+
+      desc "Force confirmation job for a specific Order Cycle"
+      task force_confirmation_job: :environment do
+        puts "WARNING: this task will process payments in customer orders"
+        exit_in_production
+
+        order_cycle_id = request_order_cycle_id
+
+        # Close Orde Cycle by moving close_at to the past
+        set_order_cycle_times(
+          order_cycle_id,
+          30.minutes.ago,
+          15.minutes.from_now
```

Comment:
Hello, shouldn't line 42 point to the "past", and close the OC? Maybe it could be `15.minutes.ago,` instead?

Reply:
Good catch! It looks like that got changed in https://github.com/openfoodfoundation/openfoodnetwork/pull/6291/commits/70d1888cc23f25176c3c297e8f0fe28186a91b13 and I missed it when reviewing.

Reply:
Ah, sorry, I didn't read the plus and minus signs properly!

Reply:
Is there a pull request already or shall I create one?

Reply:
I tried pushing to the original branch on Luis' fork, but then didn't have the permissions... I left the suggested change there instead of creating a whole new branch, but that'll probably be faster to just create a new one.

Reply:
https://github.com/openfoodfoundation/openfoodnetwork/pull/6541